tests/internal/return_true.in

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Low priority, but is there a way we can make this work on Windows too? E.g. using a simple batch script?